src/main/java/ro/itschool/ema/controllers/EventController.java

```diff
@@ -0,0 +1,40 @@
+package ro.itschool.ema.controllers;
+
```

Comment:
I have added the Participant Event Registration Endpoint to the StaffController. Should I add it to this class instead?

Reply:
no, the participant registration will be done from the ParticipantController.